provider/docker_test.go

```diff
@@ -149,12 +157,15 @@ func TestDockerGetFrontendRule(t *testing.T) {
 		},
 	}
 
-	for _, e := range containers {
-		dockerData := parseContainer(e.container)
-		actual := provider.getFrontendRule(dockerData)
-		if actual != e.expected {
-			t.Fatalf("expected %q, got %q", e.expected, actual)
-		}
+	for containerID, e := range containers {
+		t.Run(strconv.Itoa(containerID), func(t *testing.T) {
```

Comment:
Indeed, I forgot this awesome feature in go...
![](http://img.pandawhale.com/95262-keyboard-puking-gif-imgur-P4nw.gif)
Thanks :+1: